benchmarks/nemo/tracer_advection/scripts/sir_kernels_trans.py

```diff
@@ -0,0 +1,22 @@
+'''
+'''
```

Comment:
Something should go here!

Reply:
Completed.

Reply:
The docstring is still empty

Reply:
As is the docstring for the trans() function